pkg/app/sensor/app.go

```diff
@@ -477,15 +478,32 @@ func getCurrentPaths(root string) (map[string]interface{}, error) {
 				return filepath.SkipDir
 			}
 
-			if info.Mode().IsRegular() &&
-				!strings.HasPrefix(pth, "/proc/") &&
-				!strings.HasPrefix(pth, "/sys/") &&
-				!strings.HasPrefix(pth, "/dev/") {
-				pth, err := filepath.Abs(pth)
-				if err == nil {
-					pathMap[pth] = nil
+			if !info.Mode().IsRegular() {
+				return nil
+			}
+
+			pth, err = filepath.Abs(pth)
+			if err != nil {
+				return nil
+			}
+
+			if strings.HasPrefix(pth, "/proc/") ||
+				strings.HasPrefix(pth, "/sys/") ||
+				strings.HasPrefix(pth, "/dev/") {
+				return nil
+			}
+
+			// TODO: Combine this logic with the similar logic in artifacts.go
+			for _, xpattern := range excludes {
```

Comment:
This must be moved up as outlined. 

Also this same check it needed in findSymlinks() at approximately line 203 of pkg/app/sensor/data_processor.go